language/compiler/ir-to-bytecode/src/context.rs

```diff
@@ -517,21 +591,15 @@ impl<'a> Context<'a> {
     pub fn declare_field(
         &mut self,
         s: StructHandleIndex,
+        sd_idx: StructDefinitionIndex,
```

Comment:
Looks good now, I hate it, but it looks good. 

Maybe add a TODO to find a way to only pass in one "identifier" for the struct. Whether it be a handle index, def index, or name